client/packages/cli/src/schemas/sfx.ts

```diff
@@ -80,3 +73,9 @@ export const SideEffectSchema = z.object({
 })
 
 export type SideEffect = z.infer<typeof SideEffectSchema>
+
+export type SideEffectAction = SideEffect["action"]
+export const SideEffectActions = {
```

Comment:
how about adding full coverage from types/abi/src/standard.rs?

pub fn standard_sfx_abi() -> Vec<(Sfx4bId, SFXAbi)> {
    vec![
        (*b"data", get_data_abi()),
        (*b"tran", get_sfx_transfer_abi()),
        (*b"tass", get_sfx_transfer_asset_abi()),
        (*b"swap", get_swap_abi()),
        (*b"aliq", get_add_liquidity_abi()),
        (*b"rliq", get_remove_liquidity_abi()),
        (*b"cevm", get_call_evm_contract_abi()),
        (*b"wasm", get_call_wasm_contract_abi()),
        (*b"cgen", get_call_generic_abi()),
    ]
}

Reply:
I'll make sure to update the schema accordingly!

Reply:
Done; see commit [5375fe1](https://github.com/t3rn/t3rn/pull/981/commits/5375fe1d6929c9775cec0af8a8d60e403ad7af64).